frontend-plugin-core/src/test/java/com/github/eirslett/maven/plugins/frontend/lib/UtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package com.github.eirslett.maven.plugins.frontend.lib;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UtilsTest {
+
+  @Test
+  public void testImplode() {
+    final String separator = "Bar";
+    final List<String> elements = new ArrayList<String>();
+
+    Assert.assertEquals("", Utils.implode(separator, elements));
+
+    elements.add("foo");
+    elements.add("bar");
+    Assert.assertEquals("foo bar", Utils.implode(separator, elements));
```

Comment:
As discussed you could do this test as -
```
Assert.assertEquals("", Utils.implode("Bar", new ArrayList<String>()));
Assert.assertEquals("foo bar", Utils.implode("Bar", new ArrayList<>(Arrays.asList("foo","bar"))));
```

Reply:
Keeping as is which I think is more readable.